packages/react/src/components/header/components/headerActionBarItem/HeaderActionBarItemButton.tsx

```diff
@@ -0,0 +1,167 @@
+import React, { cloneElement, forwardRef, ReactNode } from 'react';
+
+import classes from './HeaderActionBarItemButton.module.scss';
+import classNames from '../../../../utils/classNames';
+import { IconAngleDown, IconAngleUp } from '../../../../icons';
+
+type ButtonAttributes = JSX.IntrinsicElements['button'];
+
+export interface HeaderActionBarItemButtonProps extends ButtonAttributes {
+  /**
+   * Id of controlled dropdown menu.
+   */
+  ariaControls?: string;
+  /**
+   * Aria label for the item.
+   */
+  ariaLabel?: string;
```

Comment:
In the coming HDS v4 code, we started a new "standard" of not to use "ariaLabel", or any "ariaProp" and use instead the actual "aria-label" etc.

This simplifies code and prop passing with `{...rest}`. I guess there is no reason to use "ariaLabel" here?

We cannot change the same thing in HeaderActionItem because that would be a braking change, but this is new component.

Reply:
good idea, but my linter doesn't like this. Have you used this notation already in somewhere?

Reply:
It should work like 'aria-label' there's examples in release-4.0.0 branch if it doesn't work for you. (Button or Tag component, don't remember which)

Reply:
Thanks, I fixed these